packages/circuit-ui/styles/style-mixins.ts

```diff
@@ -115,6 +118,16 @@ export const spacing = (
   };
 };
 
+/**
+ * @private
+ */
+export const shadow = () => (_args: ThemeArgs): SerializedStyles => css`
+  box-shadow: 0 3px 8px 0 rgba(0, 0, 0, 0.2);
```

Comment:
Should this be in design tokens, in case teams need to use a shadow in custom components?

Reply:
The future of shadows in the design system is still unclear to me. Perhaps we can turn it into a design token in the future, but for now, I'd wait and see.